### PR TITLE
Update docs regarding symbols location

### PR DESCRIPTION
**Pull request by AlanWong-MS**
### Description
Update docs regarding symbols location

### CODE REVIEW

File: _posts/2015-06-01-10-01-debug-webapi-source.md [hunk 1]
```diff
@@ -9,14 +9,17 @@ If you want to debug **OData Lib, WebAPI, Restier** source, open `DEBUG` -> `Opt
 
 1. Uncheck `Enable Just My Code (Managed only)`.
 2. Uncheck `Enable .NET Framework source stepping`.
-3. UnCheck `Require source files to exactly match the original version`.
-4. Check `Enable source server support`.
```
Comment (xuzhg) [inline _posts/2015-06-01-10-01-debug-webapi-source.md]: Enable Source Link support

**Comment by AlanWong-MS** [comment]
@xuzhg any other comments?

File: _posts/2015-06-01-10-01-debug-webapi-source.md (hunk 1 above)
Comment (AlanWong-MS) [reply _posts/2015-06-01-10-01-debug-webapi-source.md]: #done

**Comment by xuzhg** [comment]
@AlanWong-MS No. I approved it. Thanks.

**Pull request merged**